pages/exhibitions/index.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <b-container data-qa="exhibitions">
-    <header class="row">
+    <div class="row">
```

Comment:
What is the reason for changing header into div? On this page and the other pages? I think it is semantically correct to use a `<header>` here.

Reply:
I'll revert this change, I thought this might break rules for only having one header landmark but it is semantically correct